### DISCUSSION TITLE
How do you work with whole codebase /w AI?

### DISCUSSION THREAD

**Discussion by desireco** [zed-industries/zed#19841] [Feature Requests]
I am probably missing something obvious. I can ask inline for suggestions, and in Assistant panel, but at least to me, there is no simple way to ask to apply changes I get in Assistant panel to code in the tab that is opened, let alone to whole database.

Does this make sense, we usually have more then single file and I would like to move methods around, refactor them.

**Comment by phil65** [comment]
By passing the /workflow context, you can get auto-applying changes from the assistant panel. (although it seemed a bit broken for me in last days)

**Reply by SomeoneToIgnore** [reply]
There were some changes on that front: https://github.com/zed-industries/zed/pull/19900

**Reply by desireco** [reply]
I somehow missed that in the docs :)

**Reply by avramukk** [reply]
Looks like its disabled now. But why? I want to have this feature improved. That's can be better even than vscode or cursor.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
